ImagesList/ImagesListViewController.swift

```diff
@@ -2,19 +2,45 @@
 import UIKit
 
 class ImagesListViewController: UIViewController {
-    
     @IBOutlet private var tableView: UITableView!
+    private let photosName: [String] = Array(0..<14).map{"\($0)"}
+    private let nameLikeButtonOn = "like_button_on"
```

Comment:
In this sprint you don't needed in this properties. If you want to use some strings as a keys -  create enum for that.